pkg/web_app/lib/src/account.dart

```diff
@@ -60,9 +60,8 @@ void setupAccount() {
   final clientId = metaElem == null ? null : metaElem.attributes['content'];
   if (clientId == null || clientId.isEmpty) {
```

Comment:
Might it not be better to set something else here?

Like requiring that `clientId == 'fake-auth-local-dev'`